pycqed/instrument_drivers/physical_instruments/QuTech_VSM_Module.py

```diff
@@ -0,0 +1,267 @@
+                               vals=vals.Numbers())
+        for channel in self.channels:
+            for mod in self.modules:
+                mod_ch_name = 'mod{m}_ch{c}'.format(m=mod, c=channel)
+                mod_ch_scpi = 'MODULE{m}:CHANNEL{c}'.format(m=mod, c=channel)
+                self.add_parameter(mod_ch_name + '_temperature',
+                                   get_cmd='TEMPERATURE:'+mod_ch_scpi+'?',
+                                   vals=vals.Numbers())
+
+    def add_marker_parameters(self):
+        # Command class marker
+        self.add_parameter('marker_source',
+                           set_cmd='MARKER:SOURCE {}',
+                           vals=vals.Enum('int', 'ext'))
+        self.add_parameter('marker_state',
+                           set_cmd='MARKER:STATE {}',
+                           vals=vals.OnOff())
+
+        for mod in self.modules:
+            mod_name = 'mod{m}'.format(m=mod)
+            mod_scpi = 'MODULE{m}'.format(m=mod)
+            self.add_parameter(mod_name + '_marker_source',
+                               get_cmd='MARKER:' + mod_scpi + ':SOURCE?',
+                               set_cmd='MARKER:' + mod_scpi + ':SOURCE {}',
+                               vals=vals.Enum('INT', 'EXT'))
+            self.add_parameter(mod_name + '_marker_state',
+                               set_cmd='MARKER:'+mod_scpi+':STATE {}',
+                               vals=vals.OnOff())
+
+        for channel in self.channels:
+            for mod in self.modules:
+                mod_ch_name = 'mod{m}_ch{c}'.format(m=mod, c=channel)
+                mod_ch_scpi = 'MODULE{m}:CHANNEL{c}'.format(m=mod, c=channel)
+                self.add_parameter(mod_ch_name + '_marker_state',
+                                   get_cmd='MARKER:'+mod_ch_scpi+':STATE?',
+                                   set_cmd='MARKER:'+mod_ch_scpi+':STATE {}',
+                                   vals=vals.OnOff())
+                self.add_parameter(mod_ch_name + '_marker_source',
+                                   get_cmd='MARKER:'+mod_ch_scpi+':SOURCE?',
+                                   vals=vals.Enum('int', 'ext'))
+
+    def add_qubit_parameters(self):
+        # Command class qubit
+        for channel in self.channels:
+            ch_name = '_ch{c}'.format(c=channel)
+            ch_scpi = ':CHANNEL{c}'.format(c=channel)
+            self.add_parameter('qubit' + ch_name + '_description',
+                               get_cmd='QUBIT'+ch_scpi+':DESCRIPTION?',
+                               set_cmd='QUBIT'+ch_scpi+':DESCRIPTION {}',
+                               vals=vals.Strings())
+            self.add_parameter('qubit' + ch_name + '_frequency',
+                               get_cmd='QUBIT'+ch_scpi+':FREQUENCY?',
+                               set_cmd='QUBIT'+ch_scpi+':FREQUENCY {}',
+                               vals=vals.Numbers())
+            self.add_parameter('qubit' + ch_name + '_led_color',
```

Comment:
what kind of strings are accepted here? basic color names? hex values? Please add a docstring.